src/PowerShellEditorServices.VSCode/Cmdlets/VSCodeHtmlContentViewCommands.cs

```diff
@@ -59,7 +59,8 @@ protected override void BeginProcessing()
                 .GetAwaiter()
                 .GetResult();
 
-            if (_showInColumn != null) {
+            if (_showInColumn != null)
```

Comment:
Not sure if there's a rule to convert `!= null` to `is not null`...

Reply:
You'd think https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0078 would do it.